homeassistant/components/fronius/sensor.py

```diff
@@ -510,6 +515,45 @@ async def async_setup_entry(
     ),
 ]
 
+OHMPILOT_ENTITY_DESCRIPTIONS: list[SensorEntityDescription] = [
+    SensorEntityDescription(
+        key="energy_real_ac_consumed",
+        name="Energy consumed",
+        native_unit_of_measurement=ENERGY_WATT_HOUR,
+        device_class=DEVICE_CLASS_ENERGY,
```

Comment:
Should go ahead and convert to the new device class and state class enums instead of the constants.

Reply:
just saw your comment from earlier about updating all the device and state classes in a future PR 😅